Jolt/Core/Profiler.inl

```diff
@@ -25,15 +25,15 @@ ProfileThread::~ProfileThread()
 
 ProfileMeasurement::ProfileMeasurement(const char *inName, uint32 inColor)	
 {
-	if (ProfileThread::sInstance == nullptr)
+	if (ProfileThread::Instance() == nullptr)
```

Comment:
These function calls are problematic as it is part of the profiling system, so a function call overhead skews the measuring results.

Reply:
I changed it so that in static builds it's a direct access (well, via an inlined function) and in shared builds it isn't. There is no way to make this work in a shared build without the indirection (as far as I can tell).